src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Rss20ItemFormatter.cs

```diff
@@ -113,14 +113,39 @@ public override bool CanRead(XmlReader reader)
             return reader.IsStartElement(Rss20Constants.ItemTag, Rss20Constants.Rss20Namespace);
         }
 
+        XmlSchema IXmlSerializable.GetSchema()
+        {
+            return null;
+        }
+
+        void IXmlSerializable.ReadXml(XmlReader reader)
```

Comment:
Explicit private

Reply:
This is from interface. The visibility of the method is public and cannot be overridden.

Reply:
Oops, my mistake